packages/document/main-doc/docs/en/configure/app/output/clean-dist-path.mdx

```diff
@@ -1,9 +1,15 @@
 ---
 title: cleanDistPath
+configName: output.cleanDistPath
 ---
 
 # output.cleanDistPath
 
-import Main from '@modern-js/builder-doc/docs/en/config/output/cleanDistPath.md';
+- **Type:** `boolean`
```

Comment:
This configuration does not support the “auto” option in Modern.js. We should not use the rsbuild documentation.